### DISCUSSION TITLE
copy and paste

### DISCUSSION THREAD

**Discussion by trinhpxl** [keiwando/toonsquid#6] [Q&A] [answered]
hello, i looked at the handbook, but i haven't seen an answer (i dont think aa ?) is there a copy and paste function ? ^_^ or do i use the workaround, like duplicating layers ? :0 

(ALSO LOVELY APP !!! IM SWITCHING FROM FLIPACLIP TO TOONSQUID FS !! thank you for making the perfect animation program omg !! i stumbled upon the youtube demo and you SOLD me to buy the app asap ! wish you luck on future support for this app ^^)

**Comment by keiwando** [comment, marked as answer]
Thank you so much! :)

Are you asking about being able to select a certain area of a pixel layer using the selection tool and then copy/cut and paste only that region? I'm planning to add those options in the near future, so yes, for now you would have to duplicate the layer and then delete everything outside of the selection etc. and then merge the layer on the one that you want to paste the selection on.

Entire layers (without selection regions being involved) can already be copied and pasted, but I assume you already know about that since those options are right next to the "duplicate" option.
https://toonsquid.com/handbook/layers/layer_list/#actions

**Comment by trinhpxl** [comment]
ahh i see then, looking forward to the update then :D ! tysm for the answer ^_^
